assets/sass/components/dashboard/_googlesitekit-subtle-notification.scss

```diff
@@ -83,11 +79,12 @@
 	.googlesitekit-subtle-notification__action {
 		align-items: center;
 		display: flex;
-		gap: $grid-gap-phone;
+		gap: 8px;
```

Comment:
I think we should not change the gap here and adhere to the designs to use a 16px gap. Instead, what do you think about using `flex-wrap: wrap` to let the buttons stack in extremely shrunk viewports?